tsn/src/bin/throughput.rs

```diff
@@ -421,15 +420,16 @@ fn send_perf(sock: &i32, pkt: &mut Vec<u8>, size: usize) {
     }
 }
 
-fn make_send_pkt(ethernet_bytes: &Vec<u8>, pkt_info: &PktInfo) -> Vec<u8> {
-    let mut pkt: Vec<u8> = ethernet_bytes.clone();
-    println!("pkt = {:0x?}", pkt);
-    let mut pkt_info_bytes = bincode::serialize(pkt_info).unwrap();
-    println!("pkt_info_bytes = {:0x?}", pkt_info_bytes);
-    pkt.append(&mut pkt_info_bytes);
-    println!("pkt = {:0x?}", pkt);
-
-    pkt
+fn make_send_pkt(pkt: &mut Vec<u8>, ethernet: &Ethernet, pkt_info: &PktInfo, size: usize) {
```

Comment:
Function name is hard to understand.
Is it for make a packet to send?

Reply:
Yes, this function gets a packet ready to be sent

Reply:
"make" and "send" both are verb it is weird name to use as a function name

Reply:
How about prep_pkt?